src/components/menu/menu.component.js

```diff
@@ -55,6 +55,7 @@ const Menu = ({ menuType = "light", children, ...rest }) => {
               menuType,
               handleKeyDown: (ev) => handleKeyDown(ev, index),
               isFocused,
+              inMenu: true,
```

Comment:
You also need to set this in `MenuFullScreen` in the other Provider, otherwise any vertical dividers in a full screen menu won't pick this up